stage1/usr_from_coreos/coreos-common.mk

```diff
@@ -11,9 +11,13 @@ $(call setup-tmp-dir,CCN_TMPDIR)
 # systemd version in coreos image
 CCN_SYSTEMD_VERSION := v229
 # coreos image version
+ifeq ($(RKT_ACI_ARCH),aarch64)
+CCN_IMG_RELEASE := 1053.2.0
+else
 CCN_IMG_RELEASE := 1032.0.0
+endif
```

Comment:
@lucab 1068.0.0 builds OK on arm64.  I'll try running it and adjust my stage1 manifest files if needed.